cpp/modules/deck.gl/core/src/shaderlib/project/viewport-uniforms.h

```diff
@@ -30,24 +30,23 @@
 
 namespace deckgl {
 
-struct alignas(lumagl::utils::kMinDynamicBufferOffsetAlignment) ViewportUniforms {
-  COORDINATE_SYSTEM project_uCoordinateSystem;
-  PROJECTION_MODE project_uProjectionMode;
-  mathgl::Vector3<double> project_uCoordinateOrigin;
-  mathgl::Vector4<double> project_uCenter;
-  double project_uAntimeridian;
-  mathgl::Vector2<double> project_uViewportSize;
-  double project_uDevicePixelRatio;
-  double project_uFocalDistance;
-  mathgl::Vector3<double> project_uCommonUnitsPerMeter;
-  mathgl::Vector3<double> project_uCommonUnitsPerWorldUnit;
-  mathgl::Vector3<double> project_uCommonUnitsPerWorldUnit2;
-  double project_uScale;
-  mathgl::Matrix4<double> project_uViewProjectionMatrix;
-  mathgl::Vector3<double> project_uCameraPosition;
-
-  bool project_uWrapLongitude;
-  mathgl::Matrix4<double> project_uModelMatrix;
+struct ViewportUniforms {
+  int uCoordinateSystem;
```

Comment:
Since we are obsessing about memory layout, and an `int` is allowed to vary depending on the [word size of the target CPU architecture](https://en.cppreference.com/w/cpp/language/types), should we use [fixed  integer types](https://en.cppreference.com/w/cpp/types/integer)?